services/order.py

```diff
@@ -0,0 +1,35 @@
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+import datetime
+
+from db.models import Ticket, Order, MovieSession
+
+
+@transaction.atomic
+def create_order(
+        tickets: list[dict],
+        username: str, date:
+        datetime = None
+) -> None:
+    user = get_user_model().objects.get(username=username)
+    order = Order.objects.create(user=user)
+    if date:
+        order.created_at = date
+    order.save()
+    for ticket in tickets:
+        movie_session = MovieSession.objects.get(pk=ticket["movie_session"])
```

Comment:
do you really need make additional request each time? why not just
```python
Ticket.objects.create(
            movie_session=ticket["movie_session"]

Reply:
in each ticket can be different movie_sessions, you can make one order with many tickets to different movies and olso to different movie_sessions